src/sources/multisrc/wqmangastream/WQMangaStreamGenerator.js

```diff
@@ -11,4 +11,5 @@ export const KolNovelScraper = new WQMangaStreamScraper(
   53,
   'https://kolnovel.com/',
   'KolNovel',
+  { reverseChapters: true, ignoredStrings: true },
```

Comment:
I meant `ignoredStrings` should be an array of strings which would be removed from the chapterText. 

If it is a source specific issue then we should add a check for sourceId instead so it doesn't affect other sources using the multiSrc generator.

Reply:
An array? The generated text is currently all under the same class value so my solution was to check for the p.(class) within the chapter and remove every instance of it. The (class) value is not fixed, so my implementation is to find out the class value required, and apply accordingly. I'm not quite sure how I'm to make this an array though.

Yes, it is source specific so i can remove it as an option and make it target the sourceId instead. I'm not exactly sure how it would affect other sources within the multiSrc since I'm using it as an `if (this.ignoredStrings)` statement.

Reply:
```suggestion
    let chapterName = loadedCheerio('.entry-title').text();
    let chapterText = loadedCheerio('.epcontent').html();

    if (sourceId === 53) {
      let ignore = loadedCheerio('.epcontent > p').next().attr('class');
      loadedCheerio(`p.${ignore}`).remove();
      chapterText = loadedCheerio('.epcontent').html();
    }
```